travis-ymls/react-router.travis.yml

```diff
@@ -0,0 +1,89 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : react-router
+# Source Repo         : https://github.com/ReactTraining/react-router
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/react-router/builds/210998601
+# Created travis.yml  : No
+# Maintainer          : Devendranath Thadi <devendranath.thadi3@gmail.com>
+#
+# Script License      : Apache 2.0
+#
+# ----------------------------------------------------------------------------
+
+language: node_js
+node_js: node
+cache: yarn
+jobs:
+  include:
+  - stage: Test
+    env: TEST_ENV=source
+    before_script:
+    # react-router CJS build is imported by react-router-config tests,
+    # so we need to build it before running react-router-config tests
+    - cd packages/react-router && BUILD_ENV=cjs yarn build && cd -
+    script: yarn test
+  - env: TEST_ENV=cjs BUILD_ENV=cjs
+    before_script: yarn build
+    script: yarn test
+  - env: TEST_ENV=umd BUILD_ENV=umd
```

Comment:
Can we add `amd64` arch just for consistency for all 3 envs ?- source , cjs , umd - @dthadi3 , rest all looks good.

Reply:
Added amd64 arch for all the 3 env's.